README.md

```diff
@@ -232,6 +223,26 @@ Definitely Typed members routinely monitor for new PRs, though keep in mind that
 
 For a good example package, see [base64-js](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/base64-js).
 
+## Definition Owners
+
+DT has the concept of "Definition Owners" which are people who want to maintain the quality of a particular module's types
+
+  - Adding yourself to the list will cause you to be notified (via your GitHub username) whenever someone makes a pull request or issue about the package.
+  - Your PR reviews will have a higher precedence of importance to [the bot](https://github.com/DefinitelyTyped/dt-mergebot) which maintains this repo.
+  - The DT maintainers are putting trust in the definition owners to ensure a stable eco-system, please don't add yourself lightly.
```

Comment:
Thanks! Hope this is going to be enough to stop people adding themselves too soon 🤞

Reply:
Yeah, I'm curious what, if anything, should be done for existing packages. In practice on `aws-lambda` I've mostly just seen that most pinged users never respond, so I'm not that concerned, but maybe they're all just quietly fuming at the spam?

Reply:
As one of those `aws-lambda` folks, I'm happy for the spam, though you seem to smash through the requests @simonbuchan before I ever get around to them. 😃

Reply:
Having an opinion had never been a problem for me! But feel free to toss in a second review, even if it's just a empty accept: I'm a bit concerned about my random quirks being gospel for such a popular package.

Reply:
Yeah, it's not much different from now than before (though maybe because PRs don't need to be delayed by us getting round to it, maybe there could be more) 

Part of our internal chat which trigger this PR covered idle module owners - IMO, I think they're fine to stay as that, but there's probably a space to include docs on how to remove yourself. It's hard though because putting that on a new PR's chat messages feels a bit weird, but maybe it could be in this message  https://github.com/DefinitelyTyped/DefinitelyTyped/pull/44942#issuecomment-631774581 as a disclosure section 

e.g. 

<details>
    <summary>Getting too many notifications? </summary>
    how to remove yourself
</details>

Reply:
Yeah, that seems reasonable.
The other case the had been a problem I've seen was people opening issues for a package getting daunted by the number of owners and either not mentioning anyone or only picking a few. It would be nice if the bot could handle this, but for now having fewer owners is a decent mitigation. But DT issues are weird in a few ways

Reply:
@orta If it remains an issue, I think it would be great if the bot could leave a danger-like comment saying “please verify that you intended to sign-up as maintainer and get pinged for each issue/PR”. I did that manually for a while and it seemed to make half the people realize what it meant and saying “oh no thanks”. I stopped doing it, though, too tiresome and I feel bad for letting a PR sit 36 hours because of back and for the across a ~12 hour time zone difference.